src/main/java/gregtech/common/metatileentities/electric/MetaTileEntityItemCollector.java

```diff
@@ -227,33 +256,98 @@ protected void setItemSuckingRange(int itemSuckingRange) {
+                        .child(IKey.lang(getMetaFullName())
+                                .asWidget()
+                                .alignX(0.0f))
+                        .child(Flow.row()
+                                .widthRel(1.0f)
+                                .coverChildrenHeight()
+                                .margin(0, 4)
+                                .child(new ButtonWidget<>()
+                                        .marginRight(5)
+                                        .size(20)
+                                        .onMousePressed(mouse -> {
+                                            int range = rangeSync.getIntValue();
+                                            if (range > 1) {
+                                                rangeSync.setIntValue(range - 1);
+                                            }
+                                            return true;
+                                        })
+                                        .overlay(IKey.str("-1")))
+                                .child(KeyUtil.lang(TextFormatting.WHITE,
+                                        "gregtech.machine.item_collector.gui.collect_range",
+                                        () -> new Object[] {
+                                                TextFormattingUtil.formatNumbers(rangeSync.getIntValue()) })
+                                        .alignment(Alignment.Center)
+                                        .asWidget()
+                                        .height(20)
+                                        .expanded()
+                                        .background(GTGuiTextures.DISPLAY))
+                                .child(new ButtonWidget<>()
+                                        .marginLeft(5)
+                                        .size(20)
+                                        .onMousePressed(mouse -> {
+                                            int range = rangeSync.getIntValue();
+                                            if (range < maxItemSuckingRange) {
+                                                rangeSync.setIntValue(range + 1);
+                                            }
+                                            return true;
+                                        })
+                                        .overlay(IKey.str("+1"))))
+                        .child(itemFilter.initUI(guiData, panelSyncManager))
+                        .child(Flow.row()
+                                .widthRel(1.0f)
+                                .marginTop(4)
+                                .coverChildrenHeight()
+                                .child(new Grid()
+                                        .alignX(0.5f)
+                                        .height(rowSize * 18)
+                                        .minElementMargin(0, 0)
+                                        .minColWidth(18).minRowHeight(18)
```

Comment:
`.minElementMargin(0, 0)`, `.minColWidth(18)`, and `.minRowHeight(18)` are not needed